src/p2p/net_peerlist.h

```diff
@@ -290,7 +290,7 @@ namespace nodetool
 
     if (anonymize)
     {
-      std::random_shuffle(bs_head.begin(), bs_head.end());
+      std::shuffle(bs_head.begin(), bs_head.end(), std::default_random_engine(crypto::rand<unsigned>()));
```

Comment:
I would switch this to:
```c++
std::shuffle(bs_head.begin(), bs_head.end(), crypto::random_device{})
```
the `std::default_random_engine` doesn't have to be a "cryptographically secure" RNG. Otherwise this should be suitable for test purposes.

Reply:
This would of course be an easy change. If this is necessary here, though, does this solve a bug in the original code? std::random_shuffle also doesn't use a cryptographically secure RNG.

Reply:
Yes, this was/is a bug in the existing code.